airbyte-integrations/connectors/source-postgres/src/testFixtures/java/io/airbyte/integrations/source/postgres/PostgresContainerFactory.java

```diff
@@ -62,7 +65,7 @@ public void withCert(PostgreSQLContainer<?> container) {
       "openssl ecparam -name prime256v1 -genkey -noout -out ca.key",
       "openssl req -new -x509 -sha256 -key ca.key -out ca.crt -subj \"/CN=127.0.0.1\"",
       "openssl ecparam -name prime256v1 -genkey -noout -out server.key",
-      "openssl req -new -sha256 -key server.key -out server.csr -subj \"/CN=localhost\"",
+      StringUtils.replace("openssl req -new -sha256 -key server.key -out server.csr -subj \"/CN={hostName}\"", "{hostName}", HostPortResolver.resolveHost(container)),
```

Comment:
nit: can we just declare an env var and use that instead of this string substitution?

Reply:
fair